test/invalid.js

```diff
@@ -199,4 +199,36 @@ describe('Direct upload of invalid files', function () {
             });
         });
     });
+
+    describe('POST /api/v1/compendium unsupported_encoding.zip (encoding: SHIFT_JIS)', () => {
```

Comment:
should have one (or more) wrongly encoded files and assertions should check the file names.